index.js

```diff
@@ -53,7 +53,10 @@ module.exports = {
           import: [
             'js/shepherd.js',
             this.theme
-          ]
+          ],
+          processTree(tree) {
```

Comment:
This needs to be inside a vendor hash as well, with include and then manually import stuff. Needs a few things to work.